package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "osrm",
-  "version": "5.12.0-roundaboutexits.1",
```

Comment:
This needs to be the same value as master. Would conflict with the actual 5.13 version bump in https://github.com/Project-OSRM/osrm-backend/pull/4571